pars-fmt/src/lib.rs

```diff
@@ -1,290 +1,8 @@
-//! Parsing our format strings.
-//!
-//! Format strings are arbitrary strings containing 'field markers', indicating
-//! the location of fields. Field markers begin with `#{` and end with `}`,
-//! and contain the name of a struct field, or the index of a tuple field.
-//!
-//! Field markers must be separated by one or more dividing characters.
-//! During parsing, we scan the input looking for these dividing strings,
-//! which are discarded; the remaining subsections are associated with the
-//! matching fields, and parsed based on the field type.
-//!
-//! # Examples
-//!
-//!```ignore
-//! // A rect, represented as a space-delimited x, y, width, height
-//! // example input: "10 11 320 104"
-//!
-//! #[pars::fmt("#{x} #{y} #{width} #{height}")]
-//! struct Rect {
-//!     x: usize,
-//!     y: usize,
-//!     width: usize,
-//!     height: usize,
-//! }
-//!
-//!```
+//! Implementation for pars functionality. This lives in its own crate
```

Comment:
`Implementation for pars non-regex string parsing`?

Reply:
yea, getting docs updated is a future todo